### PR TITLE
Add trace extension for PHP-{54,55,56,70}

### PR DESCRIPTION
**Pull request by Gasol**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----

https://pecl.php.net/package/trace

Resolve Qihoo360/phptrace#9

### CODE REVIEW

File: Formula/php54-trace.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php54Trace < AbstractPhp54Extension
+  init
+  desc "Concise description of example package"
```
Comment (javian) [inline Formula/php54-trace.rb]: Please remove these `desc` lines for the `example package` in all files.

File: Formula/php55-trace.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php55Trace < AbstractPhp55Extension
+  init
+  desc "Concise description of example package"
```
Comment (javian) [inline Formula/php55-trace.rb]: Please remove this.

File: Formula/php56-trace.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php56Trace < AbstractPhp56Extension
+  init
+  desc "Concise description of example package"
```
Comment (javian) [inline Formula/php56-trace.rb]: Please remove this line.

File: Formula/php70-trace.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+require File.expand_path("../../Abstract/abstract-php-extension", __FILE__)
+
+class Php70Trace < AbstractPhp70Extension
+  init
+  desc "Concise description of example package"
```
Comment (javian) [inline Formula/php70-trace.rb]: Please remove this line

**Comment by Gasol** [comment]
Just remove duplicate desc

**Comment by javian** [comment]
Thanks for the submission @Gasol !

**Comment by javian** [comment]
@Gasol do you know if it also works with PHP 7.1 ?

**Comment by Gasol** [comment]
@javian No, See https://github.com/Qihoo360/phptrace/issues/71 https://github.com/Qihoo360/phptrace/issues/72

**Comment by monque** [comment]
Thanks @Gasol, PHP 7.1 will be supported in next month.

**Comment by monque** [comment]
@javian @Gasol phptrace now works with PHP 7.1

**Pull request closed without merging**